src/test/java/io/zeebe/tasklist/Demo.java

```diff
@@ -39,7 +41,7 @@ public static void main(String[] args) {
                         .zeebeTaskHeader("taskForm", getCustomTaskForm()))
             .done();
 
-    client.newDeployCommand().addWorkflowModel(workflow, "demoProcess.bpmn").send().join();
+    client.newDeployCommand().addProcessModel(workflow, "demoProcess.bpmn").send().join();
```

Comment:
:wrench: The demo doesn't work anymore. 

The client should be created with a slightly different config: 
```
final ZeebeClient client = ZeebeClient.newClientBuilder().usePlaintext().build();
```

And the process model should use user tasks instead of service tasks.

But we could fix this part later.